### PR TITLE
Deprecate `obb` command and make `obo` use the file descriptor ##bin

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This command was marked as deprecated for enough time

**Test plan**

Check if its not misused internally via rcore_cmd

**Closing issues**

dunno

### CODE REVIEW

File: libr/core/cmd_open.c [hunk 1]
```diff
@@ -334,6 +323,18 @@ static void cmd_open_bin(RCore *core, const char *input) {
 			value = input[2] ? input + 2 : NULL;
 		}
 		break;
+#if 0
```
Comment (ret2libc) [inline libr/core/cmd_open.c:326]: remove

**Comment by trufae** [comment]
The thing is: we can raise the fd and the bfd by specifying the file-descriptor number or the binfile id.

* ob raises by binid
* obo raises by io-fd

the deprecation message was because the feature was implemented already and there was two commands doing the same. but obb was more about rbin, and obo more about io, imho, so after thinking a bit i think this is better approach.

I would like to know what do you think despite the fact of the DEPRECATE message was here or there, as well as get feedback on everyone on the community who can share their thoughs on this change

**Comment by ret2libc** [comment]
Ok, seems to make sense!

**Comment by trufae** [comment]
it's ready to merge imho

**Pull request merged**